Hardware selection criteria (#36)
btcven/locha: README.md

<br>

> 

Reviewer: Unrelated changes.

```suggestion
```

## Getting started

> In this page you will find all the information related to the progress of the project, mainly in two diferents languages,[Spanish](./documentation/ES/implementacion.MD), and [English](./documentation/EN/implementation.MD). This work are still under development stage, but all day we are uodating the information to expose all the progress about the project.

Reviewer: ```suggestion
In this page you will find all the information related to the progress of the project, in two languages, [Spanish](./documentation/ES/implementacion.MD), and [English](./documentation/EN/implementation.MD). This work is still under development stage, but all days we are updating the information to expose all the progress about the project.
```

described
[here](https://github.com/btcven/locha/blob/master/documents/turpial-description.md).

<figure>
  <img src="images/turpial-finished-board.jpg" alt="drawing" height="533" width="400" align="center" />
  <figcaption>Turpial Board</figcaption>
</figure>

<figure>
  <img src="images/turpial-size-compared.jpg" alt="drawing" height="533px" width="400px" align="center" />
  <figcaption>Size comparison of a Turpial board</figcaption>

> </figure>

Reviewer: ```suggestion
<figure align="center">
  <img src="images/turpial-finished-board.jpg" alt="drawing" height="533" width="400" />
  <figcaption>Turpial Board</figcaption>
</figure>

<figure align="center">
  <img src="images/turpial-size-compared.jpg" alt="drawing" height="338px" width="600px" />
  <figcaption>Size comparison of a Turpial board</figcaption>
</figure>
```

This aligns the figures to the center and gives the correct size.

Reviewer: I haven't tested it

Reviewer: But the size is correct, alignment is unknown
